### PR TITLE
Feature/html

### PR DESCRIPTION
**Pull request by SabotageCat**
# Review for merge to Develop branch

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+  <!-- FOOTER SECTION -->
+  <footer>
+    <h4>For additonal information concerning travel and COVID-19, please visit the <a href="https://www.cdc.gov/coronavirus/2019-ncov/travelers/index.html" target="_blank">CDC Travel Guidelines site.</a></h4>
+  </footer>
+
+
+
+
+<!--
```
Comment (gmmarcoux) [inline index.html]: This is incomplete, so the js file won't be useable
""

**Review by alu1868 — APPROVED** [review]

**Review by gmmarcoux — CHANGES_REQUESTED** [review]
Everything looks great! I only had one minor change and it was just a clean up). Thank you!

File: index.html (hunk 1 above)
Comment (SabotageCat) [reply index.html]: fixed!

**Review by gmmarcoux — APPROVED** [review]
Everything else looks great! We might want to consider flipping the past search history side bar and the current search section in the HTML file for continuity (current -> past) sake, but I don't think that it's really all that important and I'm not sure if that would change the layout at all. We could talk about it tomorrow, but this looks good to go!

**Pull request merged**